Components/Modal/index.html

```diff
@@ -0,0 +1,157 @@
+              style="
+                width: 100%;
+                height: 70vh;
+                border: 0;
+                transform: scale(1);
+                overflow: hidden;
+              "
+              sandbox="allow-scripts allow-same-origin"
+            >
+            </iframe>
+          </div>
+        </div>
+      </main>
+    </div>
+    <script>
+      const Modal = ({ modalID, modalBtnID, closeModalID }) => {
+        const modal = document.getElementById(modalID);
+        const modalBtn = document.getElementById(modalBtnID);
+        const closeModal = document.getElementById(closeModalID);
+        modalBtn.onclick = () => {
+          modal.style.display = "flex";
+          setTimeout(() => {
+            modal.children[0].classList.add("kash-modal-animate");
+          }, 100);
+        };
+        closeModal.onclick = () => {
+          modal.style.display = "none";
+          setTimeout(() => {
+            modal.children[0].classList.remove("kash-modal-animate");
+          }, 100);
+        };
+      };
+      Modal({
+        modalID: "kash-modal",
+        modalBtnID: "kash-modal-open",
+        closeModalID: "kash-modal-close",
+      });
+      Modal({
+        modalID: "kash-modal-1",
+        modalBtnID: "kash-modal-open-1",
+        closeModalID: "kash-modal-close-1",
+      });
```

Comment:
You can add this script in a js file, it will be easier to maintain and reuse.

Reply:
noted thanks